src/Entity/ReferentTag.php

```diff
@@ -165,4 +175,9 @@ public function getDepartmentCodeFromCirconscriptionName(): ?string
     {
         return $this->isDistrictTag() ? \substr($this->code, 6, 2) : null;
     }
+
+    public function getZone(): Zone
```

Comment:
do we need to allow `null`?
`: ?Zone`

Reply:
the idea is to make sure there are no orphaned `ReferentTags`, so they *MUST* point to some `Zone`